ginga/util/catalog.py

```diff
@@ -4,35 +4,34 @@
 # This is open-source software licensed under a BSD license.
 # Please see the file LICENSE.txt for details.
 #
-from __future__ import print_function
+from __future__ import division, print_function
+
 import os.path
 import tempfile
 import re
 import urllib
+import time
+
 import ginga.util.six as six
+from ginga.misc import Bunch
+from ginga.util import wcs
+
 if six.PY2:
     from urllib2 import Request, urlopen, URLError, HTTPError
 else:
     # python3
     from urllib.request import Request, urlopen
     from urllib.error import URLError, HTTPError
-import time
-
-from ginga.misc import Bunch
-from ginga.util import wcs
-
-
-## star_attrs = ('name', 'ra', 'dec', 'ra_deg', 'dec_deg', 'mag', 'preference',
-##               'priority', 'flag', 'b_r', 'dst', 'description')
 
+# star_attrs = ('name', 'ra', 'dec', 'ra_deg', 'dec_deg', 'mag', 'preference',
+#               'priority', 'flag', 'b_r', 'dst', 'description')
 
-# Do we have astropy.vo installed?
-have_astropy = False
+# Do we have astroquery >=0.3.5 installed?
+have_astroquery = False
 try:
-    from astropy.vo.client import conesearch
     from astropy import coordinates, units
-    have_astropy = True
-
+    from astroquery.vo_conesearch import conesearch
+    have_astroquery = True
 except ImportError:
```

Comment:
By adding here:
```python
    from astropy.vo.client import conesearch
```
we can support older versions of `astropy`.  I think it is too early to require v2.0.